Modules/Core/Transform/include/itkv3Rigid3DTransform.h

```diff
@@ -124,8 +124,7 @@ class ITK_TEMPLATE_EXPORT Rigid3DTransform : public itk::Rigid3DTransform<TParam
   InverseTransformBasePointer
   GetInverseTransform() const override
   {
-    Pointer inv = New();
-    return this->GetInverse(inv) ? inv.GetPointer() : nullptr;
+    return Superclass::InvertTransform(*this);
   }
```

Comment:
Interestingly, this one is implemented inline, in the "h" file, whereas the other `GetInverseTransform()` member functions are implemented outside of their class definition, in their "hxx" file. Personally it's fine by me to have it inline, in the "h" file.